api/src/main/java/edu/wpi/first/shuffleboard/api/util/NetworkTableUtils.java

```diff
@@ -26,6 +26,7 @@
   private static final Pattern newMetadataPattern = Pattern.compile("/\\.");
 
   private NetworkTableUtils() {
+    throw new UnsupportedOperationException("This is a utility class!");
```

Comment:
It prevents instantiation from within the class. Useful from a maintainability standpoint